api/admin.py

```diff
@@ -102,8 +103,29 @@ def get_listings(status: str = 'all'):
             code = 200
 
     # If NOT in admin session, deny access:
+    else:
```

Comment:
Good addition, we don't want anyone besides admins to be able to get this information